### DISCUSSION TITLE
Produce still images - Out of Vram Memory

### DISCUSSION THREAD

**Discussion by jag2x** [NVlabs/instant-ngp#1206] [Ideas / Feature requests]
Hi,
I have loaded DSLR images into NERF and am rendering the same resolution out via the video rendering. The issue is, i'm running out of VRAM on my 12Gb card. I don't want to reduce the resolution if it can be helped.

Can I suggest for a future update, an option be had where the user can have extracted still image shots based on the Nerf camera, and not a video rendered file.
So if selecting still image shots, it renders a frame, based on the camera trajectory, stores the image in a folder, clears the memory of that render, then goes to the next frame, renders, stores the image, clears the memory, so on...
I'm guessing the clearning of the memory is not a trivial thing? 

Anyhow thought I'd put this up here as a future request if possible.

Thanks to everyone who put this whole thing together its awesome :)
Jag2x

**Comment by michaelrubloff** [comment]
Hey! Not sure if you'd find this helpful or not, but I created a [VRAM calculator](https://neuralradiancefields.io/vram-calculator/) because I kept having the same exact problem.
